notebooks/guides/getting-started/v2/tutorial_pt7_v2.ipynb

```diff
@@ -45,14 +45,23 @@
     "To get started, first we need to install the `cohere` library and create a Cohere client."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install cohere==5.11.3"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "\n",
-    "# pip install cohere\n",
+    "\n",
```

Comment:
can we remove the two newlines at the top of this cell? just for presentation purposes :)